util/find-doc-nits

```diff
@@ -808,8 +808,14 @@ sub checkflags {
             }
             next;
         }
-        next unless /\[B<-([^ >]+)/;
-        my $opt = $1;
+        my $opt;
+        if ( /\[B<-([^ >]+)/ ) {
+            $opt = $1;
+        } elsif ( /^B<-([^ >]+)/ ) {
+            $opt = $1;
+        } else {
+            next;
+        }
```

Comment:
You could just have made this change:

``` diff
-        next unless /\[B<-([^ >]+)/;
+        next unless /\[?B<-([^ >]+)/;
```

Reply:
Actually I would prefer if you did

Reply:
You're missing the "^" anchor.

Reply:
`/(?:^|\[)B<-([^ >]+)/`

Reply:
Is it just me - this looks like your swearing!

https://en.wiktionary.org/wiki/grawlix

Reply:
Yeahwell...

Reply:
I prefer it as-is because it is more understandable to a wider audience.

Reply:
I'm going to mark this as resolved since the simpler code is easier for more people to understand.  Unresolve if you think it important to continue this thread.